mfthomps/CyberCIEGE-port-to-Unity3d
# Issue/95 attack log format

updated format of Attack Log Display

to test

- open the Introductory Scenario and load Phase 1 (or complete until Phase 1 has been completed)
- make sure the game is in play mode
- click on the attack log button to open the attack log
- notice the attack log item is flashing red
- close the attack log with the X button
- reopen the attack log and notice the attack log item is no longer flashing

I did not add a graphic to the Attack Log button that indicates there are new attacks. I was going to handle that on another issue, because I want to make sure I'm handling the events the same way as the view tabs. I'll consult with Eric about that.
